src/EnergyPlus/NodeInputManager.cc

```diff
@@ -1141,8 +1145,21 @@ namespace NodeInputManager {
 						}
 					}
 				}
+				if ( EMSManager::CheckIfNodeMoreInfoSensedByEMS( iNode, "System Node Wetbulb Temperature" ) ) { 
```

Comment:
@Myoldmopar Style question - do we have a preference for new code for "using" verses full reference?

EMSManager::CheckIfNodeMoreInfoSensedByEMS( )
vs
using EMSManager
...
CheckIfNodeMoreInfoSensedByEMS( )